Cargo.toml

```diff
@@ -46,8 +10,4 @@ members = [
 [patch.crates-io]
```

Comment:
I actually never saw this. I do not really understand why this is necessary. Anyone knows something?

Reply:
The reason why we use [the `[patch]` section](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section) is so that we can have fully-versioned dependencies like `uefi-macros = "0.8.0"` in the various `Cargo.toml` files, yet cargo will use the local versions of these crates when building the project.

Otherwise, we'd have to write `uefi-macros = "../path/to/uefi/macros"` everwhere, manually change them to the released versions when publishing new versions of the crates, then switch them back.